web/controllers/rpi/index.js

```diff
@@ -235,6 +237,21 @@ function handleCb(request, reply) {
     request.seneca.act(senecaRegisterMsg, callback);
   };
 
+  const returnToUrl = (request, returnTo) => {
+    try {
+      const url = new URL(returnTo)
+      const permittedHosts = ['localhost:3000', 'coderdojo.com']
```

Comment:
should we include staging host too?

Reply:
Yeah ok (even though this won't work exactly as planned in Staging)